examples/externalProfileDemo.html

```diff
@@ -0,0 +1,337 @@
+<!DOCTYPE html>
+<html>
+    <!--
+        Demo to illustrate the myClientApp.contacts.showExternalContactProfile(...) and
+          myClientApp.contacts.showExternalOrganizationProfile(...) SDK methods.
+
+        Note:  Although this example is hosted with the others here on github; it won't work without
+          configuring an OAuth Client and hosting it elsewhere. See js code below for details.
+
+        Reminder: This example is using CDNs and simple javascript to illustrate the SDK. Both the
+          PureCloud SDK and Apps SDK can be obtained via npm and support many build tools
+          and javascript libraries to fit within your development environment.
+    -->
+    <head>
+        <meta charset="utf-8">
+        <title>PureCloud client-app-sdk External Contacts Demo</title>
+
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" integrity="sha512-dTfge/zgoMYpP7QbHy4gWMEGsbsdZeCXz7irItjcC3sPUFtf0kuFbDz/ixG7ArTxmDjLXDmezHubeNikyKGVyQ==" crossorigin="anonymous">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap-theme.min.css" integrity="sha384-aUGj/X2zp5rLCbBxumKTCw2Z50WgIr1vs/PFN4praOTvYXWlVyh2UtNUU0KAUhAX" crossorigin="anonymous">
+
+        <script src="https://sdk-cdn.mypurecloud.com/javascript/52.1.1/purecloud-platform-client-v2.min.js"></script>
+        <script src="https://sdk-cdn.mypurecloud.com/client-apps/1.3.0/purecloud-client-app-sdk-43593954923a1eb6031c0828e0d4d5d1.min.js"></script>
```

Comment:
As part of another PR, I added support for multiple files.  So, if you want, you can move your js and css into separate files and get proper syntax highlighting.

It still doesn't support a build, but separate files is a step in the right direction.